packages/core-components/src/lib/unsorted/viz/DataTable.svelte

```diff
@@ -434,6 +434,34 @@
 												)}
 											{/if}
 										</a>
+									{:else if column.contentType === 'delta' && row[column.id] !== undefined}
+										<div
+											class="m-0 text-xs font-medium font-ui"
+											style={`color:${
+												(row[column.id] >= 0 && !column.downIsGood) ||
+												(row[column.id] < 0 && column.downIsGood)
+													? 'var(--green-700)'
+													: 'var(--red-700)'
+											}`}
+										>
+											<div style="text-align: right;">
+												<span>
+													{formatValue(
```

Comment:
Do you think users might like the ability to use _just_ the triangle indicator as well? (Even more compact way to scan data)

So you'd be able to have a column with just triangles pointing up or down?

Could be a `showValue` prop, for example

Reply:
I like that idea - I've just added it as `showValue`. I wonder if the props that are specific to a content type should have a prefix for the content type (E.g., `deltaShowValue`). I'm not sure if that's clearer or more confusing